api/types/types.go

```diff
@@ -657,15 +657,16 @@ type Runtime struct {
 	Path string   `json:"path"`
```

Comment:
Curious; your example shows a runtime configuration that _ONLY_ has `shim.binary` set (but no `path`); is that a valid configuration? 

- If so (and the `path` is optional), we should consider adding an `omitempty` to this field.
- If that means that at least `path` _and/or_ `shim.binary` must be specified, then we should probably have a validation for that (error if none are set?).

Reply:
Thank you for the review, @thaJeztah 
The `path` is used for configuring OCI runtime for `io.containerd.runc.v2` shim, for other shims, they may or may not need the `path` property, so I added the `omitempty` attribute.